### PR TITLE
Refactor/change method names

### PR DESCRIPTION
**Pull request by catlynbowles**
What does this PR do?
- Adds sad path testing to Hydration-test and Sleep-test
- Implements meta tags for accessibility
- modified methods in Sleep and Hydration class to account for sad paths. 

Where should your reviewer start?
- Line 4 in index.html
- Line 10 in Hydration.js
- Line 7 in Sleep.js 
- Line 14 in apiCalls .js
- Line 71 in Hydration-data.js
- Line 109 in sleep-data.js
- Line 29 on Hydration-test.js
- Line 13 in sleep-test.js

What (if any) features are you implementing?
- N/A

What (if anything) did you refactor?
- Refactored tests. 

Were there any issues that arose?
- N/A

Is there anything that you need from your teammate?
- N/A

Any other comments, questions, or concerns?
- N/A

### CODE REVIEW

File: test/Sleep-test.js [hunk 1]
```diff
@@ -230,6 +320,42 @@ describe('Sleep', () => {
     expect(sleep.acquireUserAvgSleepQuality(sleepData2)).to.equal(3.0);
   });
 
+  it('should be able to check if user data is null', () => {
```
Comment (irmakerdem) [inline test/Sleep-test.js:323]: Make this specific to "hours" in description

File: test/Sleep-test.js [hunk 2]
```diff
@@ -230,6 +320,42 @@ describe('Sleep', () => {
...
+      "date": null,
+      "hoursSlept": null,
+      "sleepQuality": null
+    }
+  ]
+    expect(sleep.acquireUserAvgSleepHours(user10Data)).to.be.a('string');
+    expect(sleep.acquireUserAvgSleepHours(user10Data)).to.equal('User is missing this data!');
+  });
+
+  it('should be able to check if user data is null', () => {
```
Comment (irmakerdem) [inline test/Sleep-test.js:341]: make this specific to "quality" in description

File: src/Sleep.js [hunk 3]
```diff
@@ -20,46 +21,54 @@ class Sleep {
...
   acquireWeeklySleepQuality(data, date) {
     let allDates = data.map(obj => obj.date);
     let dateIndex = allDates.indexOf(date);
     let wholeWeek = data.slice([dateIndex - 6], [dateIndex +1]);
-    let final = wholeWeek.map(day => {
+    let finalDates = wholeWeek.map(day => {
       return {[day.date]: day.sleepQuality}
     });
-    return final;
+    if (finalDates.length === 0) {
```
Comment (ASands17) [inline src/Sleep.js:66]: Nice work adding this sad path testing! 😔

File: test/Sleep-test.js [hunk 4]
```diff
@@ -165,56 +179,132 @@ describe('Sleep', () => {
...
+    expect(sleep.acquireDailySleepQuality(sleepData1, "2022/06/15")).to.be.a('string');
+    expect(sleep.acquireDailySleepQuality(sleepData1, "2022/06/15")).to.equal('Invalid date!');
+  });
+
+  it('should be able to test whether the date is valid and also return a message if it not', () => {
+    expect(sleep.acquireWeeklyHoursSlept(sleepData1, "2022/06/04")).to.be.a('string');
+    expect(sleep.acquireWeeklyHoursSlept(sleepData1, "2022/06/04")).to.equal('Invalid date!')
+  });
+
+  it('should be able to test whether the date is valid and also return a message if it not', () => {
```
Comment (irmakerdem) [inline test/Sleep-test.js:204]: make this specific to quality in description

File: test/Sleep-test.js [hunk 5]
```diff
@@ -165,56 +179,132 @@ describe('Sleep', () => {
...
+  });
+
   it('should be able to acquire sleep quality for a specific date for a specific user', () => {
     expect(sleep.acquireDailySleepQuality(sleepData1, "2019/06/15")).to.be.a('number');
     expect(sleep.acquireDailySleepQuality(sleepData1, "2019/06/15")).to.equal(2.2);
     expect(sleep.acquireDailySleepQuality(sleepData2, "2019/06/16")).to.be.a('number');
     expect(sleep.acquireDailySleepQuality(sleepData2, "2019/06/16")).to.equal(3.5);
   });
 
+  it('should be able to notify user if date is invalid', () => {
```
Comment (irmakerdem) [inline test/Sleep-test.js:194]: make this specific to "hours" in description

File: dist/index.html [hunk 6]
```diff
@@ -1,10 +1,14 @@
...
 <html>
   <head>
+    <html lang="en">
     <meta charset="UTF-8">
+    <title>FitLit</title>
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Patrick+Hand&display=swap" rel="stylesheet"> 
+    <link href="https://fonts.googleapis.com/css2?family=Patrick+Hand&display=swap" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ASands17) [inline dist/index.html:10]: I love that you added this for accessibility! ⭐️

File: src/Sleep.js [hunk 7]
```diff
@@ -74,6 +83,9 @@ class Sleep {
       totalQuality += currentUser.sleepQuality;
       return totalQuality;
     }, 0);
+    if (totalSleepQuality === 0) {
```
Comment (irmakerdem) [inline src/Sleep.js:86]: Cool way to check for data validity!! 🤠

File: src/Sleep.js [hunk 8]
```diff
@@ -20,46 +21,54 @@ class Sleep {
...
+    return finalDates;
   }
-//acquireSleepQualityEachDayForAWeek
+
   acquireWeeklySleepQuality(data, date) {
     let allDates = data.map(obj => obj.date);
     let dateIndex = allDates.indexOf(date);
     let wholeWeek = data.slice([dateIndex - 6], [dateIndex +1]);
-    let final = wholeWeek.map(day => {
+    let finalDates = wholeWeek.map(day => {
```
Comment (irmakerdem) [inline src/Sleep.js:63]: Nice! Making the variable more descriptive! 🔎

**Review by ASands17 — COMMENTED** [review]
Great job! I think our refactor is looking really good so far.

**Review by irmakerdem — APPROVED** [review]

**Pull request merged**